okhttp/src/main/java/okhttp3/Response.kt

```diff
@@ -269,7 +269,7 @@ class Response internal constructor(
       this.code = code
     }
 
-    open fun message(message: String) = apply {
+    open fun message(message: String?) = apply {
```

Comment:
The field in Response is Nullable.  StatusLine toString assumes nullable.  So happy to discuss in the review.  But we would previously allow a java client to provide null here, and now we are stricter because of Kotlin checks...